fe/fe-core/src/main/java/com/starrocks/common/util/NetUtils.java

```diff
@@ -60,4 +61,10 @@ public static boolean isPortUsing(String host, int port) throws UnknownHostExcep
         }
         return flag;
     }
+
+    public static boolean validIPAddress(String hostname) {
+        String chunkIPv4 = "([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])";
```

Comment:
It's a bit strange to use regular expression this way. I suggest you split it to four parts check if the digit <= 255.